pasta_eln/inputOutput.py

```diff
@@ -191,6 +195,9 @@ def processPart(part:dict[str,str]) -> int:
             backend.useExtractors(fullPath, doc['shasum'], doc)
           else:
             logging.warning('  could not read file from zip: %s',part['@id'])
+        else:
+          shasum = generic_hash(Path(elnID), forceFile=True)
```

Comment:
Why doc['shasum'] is missing in this scenario and have to be generated?

Reply:
Good catch. I initially thought it would be needed but then it is not needed in the end.